tests/test_testtools.py

```diff
@@ -36,3 +36,12 @@ def test_exception_is_raised(self):
         self.assertThat(mocked.method(), Equals(1))
         with suppress(exceptions.MethodCallError):
             flexmock_teardown()
+
+    def test_flexmock_teardown_works_with_testtools_part1(self):
+        flexmock().should_receive("method1").ordered()
+
+    def test_flexmock_teardown_works_with_testtools_part2(self):
+        mock = flexmock().should_receive("method2").ordered().mock()
+        # Raises CallOrderError if flexmock teardown is not automatically called
+        # after test part 1 above
```

Comment:
Do you have a way to test this ?

Reply:
Test what exactly? If this test doesn't fail, everything is working.

Instead these separate teardown tests for each integration, in #114 I'm planning to run all the tests with all the test runners.

Reply:
To test the test. Make the teardown not work and check that the test is working as intended :)

Reply:
Ah, you mean testing manually if this works. I was thinking why you want to make a test for testing a test :laughing: You can remove the integration hook from `integrations.py` and then this test should fail.